addons/flags/XEH_missionDisplayLoad.sqf

```diff
@@ -0,0 +1,17 @@
+#include "script_component.hpp"
+
+if (!hasInterface) exitWith {};
+
+params ["_display"];
+
+_display displayAddEventHandler ["MouseZChanged", {
+    params ["", "_scroll"];
+    [_scroll] call FUNC(handleScrollWheel);
+}];
+
+_display displayAddEventHandler ["MouseButtonDown", {
+    params ["", "_button"];
+    if (GVAR(isPlacing) isNotEqualTo PLACE_WAITING) exitWith {false};
+    if (_button isNotEqualTo 1) exitWith {false}; // 1 = Left mouse button
```

Comment:
What about those who set the right mouse button to be the left one in the OS mouse settings ?

Reply:
I don't know how it will be registered ingame. I assume this will be fine because [fortify](https://github.com/acemod/ACE3/blob/master/addons/fortify/XEH_missionDisplayLoad.sqf) and [explosives](https://github.com/acemod/ACE3/blob/master/addons/explosives/functions/fnc_cancelPlacement.sqf) do the same thing.